tests/gtest/are_images_equal.cc

```diff
@@ -7,7 +7,8 @@
 #include "aviftest_helpers.h"
 #include "avifutil.h"
 
-using libavif::testutil::AvifImagePtr;
+using ::libavif::testutil::AreImagesEqual;
+using ::libavif::testutil::AvifImagePtr;
```

Comment:
Nit/optional: Just wondering why the root namespace :: is used with the libavif namespace in these two using statements. We don't use the root namespace :: with the std namespace in this file, for example.

Reply:
It's recommended for "using" statements, e.g. in https://abseil.io/tips/119 "When declaring namespace aliases and using-declarations, use fully qualified names (with leading `::`) unless you are referring to a name inside the current namespace."